povm_toolbox/quantum_info/multi_qubit_dual.py

```diff
@@ -85,6 +85,7 @@ def _build_dual_operators(
 
         Args:
             frame: The primal frame from which we will build dual operators.
+            alphas: TODO.
```

Comment:
@timmintam can you please make a suggestion for what to write here?

Reply:
I will update those directly in #21 as the overall dual workflow is being refactored. But fyi, I would write something along the lines "parameters of the frame super-operator used to build the dual operators."